addons/mass_mailing/views/mailing_contact_views.xml

```diff
@@ -156,14 +156,15 @@
         <field name="priority">10</field>
         <field name="arch" type="xml">
             <form string="Mailing List Contacts">
+                <field name="write_date" invisible="1"/>
```

Comment:
We generally use id field instead (id = False). Indeed write date not being set is more a side effect, using id is clearer.